<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agenda</class>
 <widget class="QMainWindow" name="agenda">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagen/logo.png</normaloff>imagen/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Buscador de contato</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="buscador">
         <property name="toolTip">
          <string>Buscador de contacto</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Apellido y nombre/Razón social/Número</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="tabla_contactos">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Contacto</string>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Número</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="agenda_completa">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Agenda Completa</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagen/agenda_completa.png</normaloff>imagen/agenda_completa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>169</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="grupo_contacto">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Nuevo contacto</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sexo/Género</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sexo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="apellido">
         <property name="placeholderText">
          <string>Apellido </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0" rowspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tipo de
documento</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="tipo_doc">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="numero">
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="placeholderText">
            <string>Número</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="email">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QGroupBox" name="grupo_rubro">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Rubro de proveedor</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="rubro">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nuevo_rubro">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Añadir nuevo rubro</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagen/signo_mas.png</normaloff>imagen/signo_mas.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tipo de contacto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tipo_contacto">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Cliente/Proveedor</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="maxVisibleItems">
            <number>12</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtBottom</enum>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="direccion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Dirección</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="telefono">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Teléfono</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Condicion frente al IVA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tipo_iva">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nombre">
         <property name="placeholderText">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="grupo_medio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Medio por el cual nos conoció</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="medio_contacto">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nuevo_medio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Añadir nuevo medio de contacto</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagen/signo_mas.png</normaloff>imagen/signo_mas.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="localidad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Localidad</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="nuevo_contacto">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Crear nuevo contacto</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/signo_mas.png</normaloff>imagen/signo_mas.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guardar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Guardar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/guardar.png</normaloff>imagen/guardar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eliminar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Eliminar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/eliminar.png</normaloff>imagen/eliminar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modificar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Modificar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/editar.png</normaloff>imagen/editar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="limpiar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Limpiar pantalla</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/limpiar.png</normaloff>imagen/limpiar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buscador</tabstop>
  <tabstop>tabla_contactos</tabstop>
  <tabstop>tipo_contacto</tabstop>
  <tabstop>tipo_iva</tabstop>
  <tabstop>apellido</tabstop>
  <tabstop>nombre</tabstop>
  <tabstop>tipo_doc</tabstop>
  <tabstop>sexo</tabstop>
  <tabstop>direccion</tabstop>
  <tabstop>localidad</tabstop>
  <tabstop>telefono</tabstop>
  <tabstop>email</tabstop>
  <tabstop>medio_contacto</tabstop>
  <tabstop>nuevo_medio</tabstop>
  <tabstop>rubro</tabstop>
  <tabstop>nuevo_rubro</tabstop>
  <tabstop>nuevo_contacto</tabstop>
  <tabstop>guardar</tabstop>
  <tabstop>eliminar</tabstop>
  <tabstop>modificar</tabstop>
  <tabstop>limpiar</tabstop>
  <tabstop>agenda_completa</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
